pom.xml

```
@@ -27,8 +20,88 @@
       <name>Zextras Artifact Manager-releases</name>
       <url>https://zextras.jfrog.io/artifactory/java-sdk</url>
     </repository>
+    <repository>
+      <id>default</id>
+      <name>Maven public</name>
+      <url>https://repo.maven.apache.org/maven2/</url>
+    </repository>
+    <repository>
+      <id>jitpack.io</id>
+      <name>Jitpack</name>
+      <url>https://jitpack.io</url>
+    </repository>
+    <repository>
+      <id>zextras-public-snapshots</id>
+      <name>Zextras public snaposhots</name>
+      <url>https://repo.zextras.io/maven/snapshot</url>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
+    <repository>
+      <id>zextras-public-release</id>
+      <name>Zextras public releases</name>
+      <url>https://repo.zextras.io/maven/release</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
   </repositories>
 
+  <pluginRepositories>
+    <pluginRepository>
+      <id>default</id>
+      <name>Maven public</name>
+      <url>https://repo.maven.apache.org/maven2/</url>
+    </pluginRepository>
+    <pluginRepository>
+      <id>jitpack.io</id>
+      <name>Jitpack</name>
+      <url>https://jitpack.io</url>
+    </pluginRepository>
+    <pluginRepository>
```

You need to add `the-lucky-gunslingers` as plugin repository
```suggestion
    <pluginRepository>
      <id>the-lucky-gunslingers</id>
      <name>Zextras Artifact Manager-releases</name>
      <url>https://zextras.jfrog.io/artifactory/the-lucky-gunslingers-java-sdk</url>
    </pluginRepository>
    <pluginRepository>
```